app/assets/javascripts/pages/errors.coffee

```diff
@@ -171,3 +185,48 @@ $('form#notify').submit ->
       $('.messageTextarea').val('')
       return
   false
+
+PubSub.subscribe('assigned.website', (ev, website)->
+  $.getJSON Routes.api_v1_website_path(website.id), (data) ->
+    $('#current-website').render data
+)
+
+$('.tab').hide()
+$('.tab2, .tab3').addClass('disabled')
```

Comment:
Dupa signup cand se adauga un website nou, folosim $.getJSON pentru a afisa app_key si app_secret la pasul 2 "Installation and configuration".
.tab2 si .tab3 apartin modalului si au clasa disabled pentru a tine userul la pasul respectiv, adica sa nu poata merge la pasul 2 pana nu adauga un website si la pasul 3 pana nu alege o platforma.